Android/app/build.gradle

```diff
@@ -15,6 +15,18 @@ try {
     keystoreProperties['storePassword'] = ''
 }
 
+// Go backend build constants
+def goToolChain = 'go1.21.1'
+def goSourceDir = "${projectDir}/src/main/go"
```

Comment:
I was checking the guideline at https://developer.android.com/studio/projects#ProjectView

The `src/main/go` aligns with `src/main/java` and `src/main/kotlin`, though I wonder if it should be `src/go`, since it's `src/cpp`.

I think the decision is based on whether we can use source sets for Go. `main` is a source set. Are we going to have different source sets in Go? Does that even make sense?

Reply:
We don't use any concepts of source sets in Go, so I would go with `src/go`, to align with `src/cpp`.